tools/mage/dev.go

```diff
@@ -265,22 +279,10 @@ func (Dev) InitStack() error {
 	if err := json.Unmarshal(jsonVal, &key); err != nil {
 		return err
 	}
-
 	if err := writeToFile(filepath.Join(devDir, "admin_api_key.txt"), []byte(key.Key)); err != nil {
 		return err
```

Comment:
I would also be fine with logging this as a warning instead of returning immediately.